### DISCUSSION TITLE
Some disk for PCEmulator

### DISCUSSION THREAD

**Discussion by FreeLANMan** [fdivitto/FabGL#282] [Show and tell]
Some disk for the PCEmulator:
https://drive.google.com/drive/folders/1OFRNx3wDCHZ4ajmrT9v_o90OhCe7HoOJ

**Comment by gnokketto** [comment]
How use this ? and its possibile add more game to VIC20 ? the site inside code (8bit) is down..

**Reply by FreeLANMan** [reply]
This is only for PCEmulator example.
You have to config the machine and add disks.

**Comment by FreeLANMan** [comment]
New disk with games!
https://drive.google.com/file/d/1kmBFcYbHhnyo3bOC0-t6ZXwnfoIhfibR/view?usp=share_link

- Games included:
Bad Street Brawler      
Crazy Digger            
Gauntlet                
Bubble Ghost            
Lode Runner             
Rampage                 
Solomon's KEY           
Death Sword (Barbarian)
La abadia del crimen  
L.A. CRACKDOWN        
Leisure Suit Larry 1  
Mixed-up Mother Goose 
Shadowgate
THE QUESTPROBE HULK                   
INDIANA JONES REVENGE OF THE ANCIENTS 
James Bond 007 in Goldfinger          
Lancelot                              
QUESTPROBE Spider-Man                 
Suspect: An Interactive Mystery
Breakers                            
The Hitchhiker's Guide to the Galaxy 
Leather Goddesses of Phobos          
A Mind Forever Voyaging              
PLANETFALL                           
Star Trek The Kobayashi Alternative  
Trinity  
Classic Concentration  
Mah Jongg solitaire    
Soko-Ban               
Tetris 1987            
Wheel Of Fortune
The Bards Tale           
BattleTech               
DND Dungeons and Dragons 
StarFlight               
Ultima I                 
Ultima III
ChessmasteR 2000                 
War in Middle Earth LotR         
OGRE (1986)                      
Patton vc Rommel                 
The Ancient Art of War at Sea
Elite                  
GATO                   
Jet 2.12 (1988)        
Star Wars (Broderbund) 
TAU-CETI
Alterego             
Super Hang-On (Sega) 
Skate or Die!        
Speedball            
KINGS of the BEACH
